lighty-modules/lighty-netconf-sb/src/test/java/io/lighty/modules/southbound/netconf/tests/SampleConfigTest.java

```diff
@@ -33,7 +33,7 @@ public void loadTopLevelSingleNodeModelsFromJsonConfig() throws Exception {
                 .getEffectiveModelContext().getModules().size();
         assertTrue(lightyController.shutdown(TIME_OUT, TimeUnit.SECONDS));
 
-        assertEquals(loadedModulesSize, 16);
```

Comment:
No, you have to adapt revisions in: `sampleConfigSingleNode.json`.

Reply:
removed this commit and changed back to original size.